landscape/how-to/install-landscape-server-on-microsoft-azure.md

```diff
@@ -0,0 +1,205 @@
+# How to install Landscape Server on Microsoft Azure
+
+This guide provides an example of how to install and set up your Landscape server on Microsoft Azure with [cloud-init](https://cloudinit.readthedocs.io/en/latest/). The instructions here can be used for both FIPS-hardened or non-hardened systems.
+
+> **For the most up-to-date documentation on Microsoft Azure, see [Microsoft Azure documentation](https://learn.microsoft.com/en-us/azure).
+
+**Contents:**
+
+- Install and set up Microsoft Azure CLI
+- Provision Azure resources and deploy
+- Deploy Landscape Server VM with cloud-init
+- Configure Landscape
+- (Optional) Perform a complete teardown
+
+
+## Install and set up Microsoft Azure CLI
+
+### Install `Azure CLI`
+
+To install `Azure CLI`, refer to the [Install Azure CLI on Ubuntu](https://documentation.ubuntu.com/azure/en/latest/azure-how-to/instances/install-azure-cli/) guide.
+
+### Connect `Azure` with your Microsoft Azure account
+
+The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+
+Run the `az login` command:
+```
+az login
+```
+
+If the Azure CLI can open your default browser, it initiates authorisation code flow and opens the default browser to load an Azure sign-in page.
```

Comment:
```suggestion
If the Azure CLI can open your default browser, it will open the default browser and load an Azure sign-in page for you to sign in with your Azure account. Otherwise, it will instruct you to open a browser page and enter the code displayed in your terminal.
```

Reply:
Just wanted to provide some additional information here.